test/api_integration/apis/saved_objects/bulk_create.js

```diff
@@ -106,6 +109,9 @@ export default function({ getService }) {
                     title: 'An existing visualization',
                   },
                   references: [],
+                  migrationVersion: {
+                    visualization: resp.body.saved_objects[0].migrationVersion.visualization,
+                  },
```

Comment:
I don't see an API integration test added for this behavior fix?

Reply:
This was a bug in the existing tests. The `expectSuccessResult` helper for bulkCreate previously expected an undefined migrationVersion, but I updated it to expect a defined migrationVersion, so this is tested. There's so much behaviour in the test mocks and assertions that it's sometimes hard to know what a test is actually asserting.

Reply:
No, I mean can't we add am API integ test for

> When bulk creating saved objects without specifying the id's of the objects to be created, the SavedObjectsClient Javascript API and HTTP API incorrectly returned the raw id in the response

?

Reply:
ah yes, that makes sense.